File: dmpWorker.go
package main

import (
	"fmt"
	"time"

	"github.com/sergi/go-diff/diffmatchpatch"
)

var dmp = diffmatchpatch.New()

const (
	text1 = "Lorem ipsum dolor."
	text2 = "Lorem dolor sit amet."
)

func createPatch(lastDoc, currDoc string) (_result string) {
	t0 := time.Now()

	diffs := dmp.DiffMain(lastDoc, currDoc, false)
	diffs = dmp.DiffCleanupSemantic(diffs)

	patch := dmp.PatchMake(diffs)

	result := dmp.PatchToText(patch)

	t1 := time.Now()
	fmt.Print(t1.Sub(t0).Nanoseconds())
	fmt.Println("ns")

	return result
}

func main() {
	fmt.Println(createPatch(text1, text2))
}

File: dmpWorker.js
var DiffMatchPatch = require('diff-match-patch');
var dmp = new DiffMatchPatch();

const text1 = "Lorem ipsum dolor.";
const text2 = "Lorem dolor sit amet.";

function createPatch(lastDoc, currDoc) {
    var t0 = process.hrtime()[1];

    var diff = dmp.diff_main(lastDoc, currDoc, false);
    dmp.diff_cleanupSemantic(diff);

    var patch = dmp.patch_make(lastDoc, diff);

    var result = dmp.patch_toText(patch);

    var t1 = process.hrtime()[1];
    console.log((t1 - t0) + 'ns');

    return result;
}

console.log(createPatch(text1, text2));